tfhe/Cargo.toml

```diff
@@ -46,7 +46,7 @@ hex = "0.4.3"
 # End regex-engine deps
 # Used for backward compatibility test metadata
 ron = "0.8"
-tfhe-backward-compat-data = { git = "https://github.com/zama-ai/tfhe-backward-compat-data.git", branch = "v0.1", default-features = false, features = [
+tfhe-backward-compat-data = { git = "https://github.com/zama-ai/tfhe-backward-compat-data.git", branch = "v0.2", default-features = false, features = [
```

Comment:
what's the branch change for ?

Reply:
because I updated some test metadata types in `tfhe-backward-compat-data` so it is not backward compatible, meaning that if we don't update the branch, we won't be able to reproduce tests on previous commits in tfhe-rs.